utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -220,7 +221,7 @@ namespace gdasapp {
 
       // Basic QC
       Eigen::Array<bool, Eigen::Dynamic, 1> boundsCheck =
-        (iodaVars.obsVal_ > sstMin && iodaVars.obsVal_ < sstMax);
+        (iodaVars.obsVal_ > sstMin && iodaVars.obsVal_ < sstMax && iodaVars.datetime_ > 0.0);
```

Comment:
what if the reference time is the middle of the window? I might be missing something but I don't think we should remove obs for which `iodaVars.datetime_ > 0.0`

Reply:
> what if the reference time is the middle of the window? I might be missing something but I don't think we should remove obs for which `iodaVars.datetime_ <= 0.0`

I believe `iodaVars.datetime_` should be positive since it has been set from epochtime, and it was also applied to trim function at AMSR2 ioda converter.

https://github.com/NOAA-EMC/GDASApp/blob/ed441fd15cb25036bd6fc1f0ffe10acc03887bc9/utils/obsproc/IcecAmsr2Ioda.h#L117